### ISSUE TITLE
:hammer: [Post-processing] If `model` is None, use dl xs as logits

### ISSUE THREAD

**Issue by o-laurent** [ENSTA-U2IS-AI/torch-uncertainty#164]
This would allow using the method without any inner model.
